### ISSUE TITLE
Pixels logo on main page hard to read with Github dark theme

### ISSUE THREAD

**Issue by mpalomas** [parasyte/pixels#155]
This is probably the lowest priority ever...

I'm using Github dark theme, and with this theme, the Pixels logo is hard to read, because it's a png with transparent background, and the black text no longer stands out enough on a dark background:

![image](https://user-images.githubusercontent.com/34667/113502652-eef48000-952d-11eb-811a-cb9b1e795dbf.png)

So maybe adding a background color, or adding contours to the text would improve readability.

**Comment by parasyte** [comment]
😄 I use the dark theme as well. I've gotten so used to the way the logo looks with this theme...

The thought comes up from time to time about whether a lighter colored background should be used, or just replace the logo with a new design. If giving it a background, I would like it to look intentional so that it can frame the logo in a presentable way. A white rectangle on a dark background would look unprofessional. But maybe it could be ok with rounded corners or some kind of border? The solution doesn't seem obvious to me.

**Comment by mpalomas** [comment]
I agree it is not as straightforward as it sounds, and I am no artist/UX design expert.
Maybe for your next release announcement you can ask Reddit (or the community at large) for help or recommendations.
The pixelated Ferris looks really cool though, this little guy should stay!

**Comment by mpalomas** [comment]
I have a "mitigation": it turns out there is also a "dimmed dark" theme, which is a nice compromise for my eyes, and the logo looks rather good on it. Really the minor annoyance only appears with the true dark theme, which is just too dark anyway. We could argue it's an "upstream" problem.

**Comment by parasyte** [comment]
I switched to the dimmed dark theme and agree it is much more pleasant in general. Also here is what the existing logo looks like on this background, for comparison:

![dimmed-dark](https://user-images.githubusercontent.com/456942/118069492-26d6ba80-b359-11eb-9067-505d24a76465.png)

I'm keeping this issue open since I believe there are improvements that can be made to the logo itself. I just don't example know what those would be right now.
